src/config/constants.ts

```diff
@@ -35,6 +35,7 @@ export const HOST_STATUS_LABELS: { [key in Host['status']]: string } = {
   insufficient: 'Insufficient',
   disabled: 'Disabled',
   installing: 'Installing',
+  'installing-in-progress': 'Installing',
```

Comment:
Don't we need to differentiate these two states? How long is normally expected to be in `installing` state (not in `installing-in-progress`). What about `Installation pending`?

Reply:
When in status Installing - get next step will return the install command
When in status installing-in-progress - it means that the installation process was started and we updated the installation progress - in this state get-next-step will not return any command.
we added this state in order to avoid starting the installation twice.

Reply:
I've as agreed in the Slack discussion, I've updated label for 'installing' to be 'Starting Installation'